### ISSUE TITLE
National characters (Polish) are entered incorrectly

### ISSUE THREAD

**Issue by ghost** [mhammond/pywin32#327]
Entering \(Polish\) national characters using standard Windows keyboard \(using so called "Polish Programmer's Keyboard"\) enters wrong Unicode characters.

OS: Win XP SP2, Polish edition.

I have tried changing fonts to what are Unicode fonts as far as I know \(Times New Roman, Arial, etc\), including fonts designated as "CE" in Windows \(Central Europe\) as well. It doesn't work.

I made sure that Polish Programmer's Keyboard is turned on which gives me correct encoding in almost all Windows applications, including Unicode editors like UniRed.

Still, Pythonwin shell in particular thinks that 
AltGr+a \(standard way of entering 'LATIN SMALL LETTER A WITH OGONEK'\) is actually 'SUPERSCRIPT ONE' for example.

To summarize:

1\. IDLE edits the text in Unicode correctly provided there's a \#-\*- coding: utf-8 -\*- header in first line.

2\. Pythonwin executes that file correctly.

3\. Pythonwin enters national characters INCORRECTLY \(at least as far as Polish is concerned, but I suspect this is also the case with other languages\).

See attached screenshot for visualisation of the problem.

Reported by: admindomeny

Original Ticket: "pywin32/bugs/327":https://sourceforge.net/p/pywin32/bugs/327
